wandb/sdk/wandb_run.py

```diff
@@ -3169,23 +3151,20 @@ def log_artifact(
         """Declare an artifact as an output of a run.
 
         Args:
-            artifact_or_path: (str or Artifact) A path to the contents of this artifact,
-                can be in the following forms:
-                    - `/local/directory`
-                    - `/local/directory/file.txt`
-                    - `s3://bucket/path`
-                You can also pass an Artifact object created by calling
-                `wandb.Artifact`.
-            name: (str, optional) An artifact name. Valid names can be in the following forms:
-                    - name:version
-                    - name:alias
-                    - digest
-                This will default to the basename of the path prepended with the current
-                run id  if not specified.
-            type: (str) The type of artifact to log, examples include `dataset`, `model`
-            aliases: (list, optional) Aliases to apply to this artifact,
+            artifact_or_path: A path to the contents of this artifact,
+                can be in the following forms
+            - `/local/directory`
+            - `/local/directory/file.txt`
+            - `s3://bucket/path`
+            name  (Optional[str]): An artifact name. Defaults to the basename of the path prepended with the current
+            run id  if not specified. Valid names can be in the following forms:
```

Comment:
There are extra spaces around `run_id` here; maybe they were backticks that got removed?